Cargo.toml

```diff
@@ -16,6 +16,10 @@ default = []
 mpio = ["mpi-sys", "hdf5-sys/mpio"]
 lzf = ["lzf-sys", "errno"]
 blosc = ["blosc-sys"]
+# The features with version numbers such as 1.10.3, 1.12.0 are metafeatures
+# and is only available when the HDF5 library is at least this version.
+# Features have_direct and have_parallel are also metafeatures and dependent
```

Comment:
@mulimoen `have_direct` and `have_parallel` and not features as of now. Should we make them features as well? I.e., `feature = "have-direct"` and `feature = "have-parallel"`, so that they show up the same way as versions?

Reply:
Oh yeah, that is of course better, I assumed they would show up, but it seems not